src/main/java/io/confluent/connect/jdbc/dialect/SqlServerDatabaseDialect.java

```diff
@@ -63,6 +67,15 @@
   private static final String DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss.SSSSSSS ZZZZZ";
   private static final DateTimeFormatter DATE_TIME_FORMATTER =
       DateTimeFormatter.ofPattern(DATE_TIME_FORMAT);
+  private static final int MSSQL_2016_VERSION = 13;
+  private static final int PRE_MSSQL_2016_VERSION = 12;
```

Comment:
I'm not sure defining this by its year of release is wise. Doesn't it make more sense to use the version number itself, instead of the year? It's more specific.

Reply:
Chatted offline about this: MSSQL Server tends to define their versions in terms of release years. Ex MSSQL Server 2016 is used to refer that version more than say MSSQL Server 13. TLDR: Keeping as is